### DISCUSSION TITLE
TeamTalk detected by Windows Defender when trying to launch the installer

### DISCUSSION THREAD

**Discussion by trancer1994** [BearWare/TeamTalk5#2264] [Q&A]
### Description

Hello,

Whenever I try to install any of the latest versions of TeamTalk QT, more specifically the client, including the beta versions, I get a "Windows protected your PC" dialog before I can even launch the installer, after which I have to press More Info and then Run Anyway to continue. I'm not sure why this is happening but it really needs to be fixed because anyone new who wants to use TeamTalk will be put off from using it because they will think it is some sort of virus. 

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [ ] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

The TeamTalk installer should launch as soon as you click on it or open it from either the downloads thing in the browser you use or the place you saved it on your computer. 

### Actual behavior

When I open the TeamTalk installer to install the program, a windows defender dialog comes up which I have to navigate through. It takes a few seconds but it's really annoying. 

### Steps to reproduce problem

1. Download either the latest beta or the latest stable version of TT from your website. 
2. 2. When it's in the downloads list and it's completed, try opening the installer by clicking on the downloaded file to run it.
3. If you have UAC enabled, it should come up with a dialog asking you whether you want to allow this app to make changes to your pc; click Yes.
4. If you don't, it should instantly come up with a dialog that says, "Windows protected your pc". Press the tab key until you hear "More info" and press enter.
5. Then, click on "Run Anyway". It will then, finally, open the installer.
6. Follow the normal set of instructions the installer gives you to install the software.

You must do this on a Windows PC that DOES NOT already have TeamTalk installed.

**Comment by bear101** [comment]
TeamTalk is no longer signed with a Code Signing Certificate. The price for a code signing certificate is several $100 per year and I don't have enough income to pay for it. macOS is also not signed for the same reason.

https://www.ksoftware.net/code-signing-certificates/

**Comment by trancer1994** [comment]
Can't you find a cheaper cert to sign it with?
